packages/web/src/components/list/MultiList.js

```diff
@@ -365,6 +378,29 @@ class MultiList extends Component {
 		});
 	};
 
+	updateDefaultQuery = (queryOptions) => {
```

Comment:
Seems like repeating coding, can we make common util function for `updateDefaultQuery`?

Reply:
cool, will do something.